### PR TITLE
Updated copy around Image Processing functions

### LINKED ISSUE #280: Normalize copy around Image Processing functions
**Is your enhancement related to a problem? Please describe.**
I received the following feedback via email:
> I believe it should be called "Use OCR to scan for text in the image" to explain what it actually does and connect back to the setting itself. The same goes for Rescan Alt Text. The setting of managing this is called "Automatically Caption Images". Not a very obvious connection. On top, you know there is actually another media meta field called Caption. To make the chance of confusion even bigger the setting is explained with "Images will be captioned with alt text upon upload". Does this mean that the Alt Text will be put as Caption as well?  It seems that the Settings dialogue and the meta box was written by two completely different departments who never spoke to each other.

**Describe the solution you'd like**
There's a solid point here where we should look at normalizing labels and helper text in different places:

**ALT TEXT**
- Setting label: `Automatically Caption Images`
- Setting helper text: `Images will be captioned with alt text upon upload`
- Threshold setting label: `Caption Confidence Threshold`
- Threshold setting helper text: `Minimum confidence score for automatically applied image captions, numeric value from 0-100. Recommended to be set to at least 75.`
- Data currently flows into the `Alternative Text` field
- Media Library image processing option label: `Scan Alt Text` and `Rescan Alt Text`

**TAGGING**
- Setting label: `Automatically Tag Images`
- Setting helper text: `Images will be tagged upon upload`
- Threshold label: `Tag Confidence Threshold`
- Threshold helper text: `Minimum confidence score for automatically applied image tags, numeric value from 0-100. Recommended to be set to at least 70.`
- Media Library image processing option label: `Generate Tags` and `Regenerate Tags`

**SMART CROPPING**
- Setting label: `Enable smart cropping`
- Setting helper text: `Crop images around a region of interest identified by ComputerVision`
- Media Library image processing option label: `Generate Smart Thumbnail` and `Regenerate Smart Thumbnail`

**OCR**
- Setting label: `Enable OCR`
- Setting helper text: `Detect text in an image and store that as post content`
- Media Library image processing option label: `Scan Text` and `Rescan Text`

**Designs**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by s3rgiosan**
### Description of the Change

Normalize copy around Image Processing functions.

Closes #280 

### Alternate Designs

N/A

### Possible Drawbacks

None identified.

### Verification Process

See [Google Doc with updated copy recommendations.](https://docs.google.com/document/d/1IrUO9C7RTTn0EZiJfwlf-lu320KanxDdVKXK2DUHT-o/edit)

### Checklist:

- [X] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Changelog Entry

```
### Changed
- Normalize copy around Image Processing functions (props @s3rgiosan)
```

### CODE REVIEW

File: includes/Classifai/Providers/Azure/ComputerVision.php [hunk 1]
```diff
@@ -860,14 +860,14 @@ public function setup_fields_sections() {
 				'input_type'    => 'checkbox',
 				'default_value' => $default_settings['enable_ocr'],
 				'description'   => __(
-					'Detect text in an image and store that as post content',
+					'OCR detects text in images (i.e., handwritten notes) or PDF files and saves that as post content.',
```
Comment (iamdharmesh) [inline includes/Classifai/Providers/Azure/ComputerVision.php]: Hi @jeffpaul,

This text update is correct as per the given Google docs. but, I think OCR setting here is only related to images as we already have separate settings for scanning/reading PDFs. Can we get it updated only for images in google docs or any suggestions? 

image

Thanks.

Comment (jeffpaul) [reply includes/Classifai/Providers/Azure/ComputerVision.php]: @iamdharmesh do you have a screenshot showing the separate settings for scanning/reading PDFs as you noted?

Comment (iamdharmesh) [reply includes/Classifai/Providers/Azure/ComputerVision.php:863]: image

**Review by iamdharmesh — APPROVED** [review]
Thanks for the great work here @s3rgiosan.
LGTM!! :+1:

**Pull request merged**